components/scream/p3/p3_impl.hpp

```diff
@@ -0,0 +1,40 @@
+#include "p3.hpp"
```

Comment:
Move this to p3.cpp when Data is converted to FortranData and stripped of its template params.